app/views/events/_form.html.erb

```diff
@@ -1,97 +1,91 @@
-<%= form_for @event, :html => { :class => "form-horizontal event" } do |f| %>
+<%= form_for @event, html:{ class: 'form-horizontal event' } do |f| %>
 
-    <%= render partial: "application/validation_errors", locals: {model: @event} %>
+    <%= render partial: 'application/validation_errors', locals: {model: @event} %>
 
     <div class="form-group">
-      <%= f.label Event.human_attribute_name(:name), :class => 'control-label col-lg-2' %>
```

Comment:
why not `human_attribute_name` anymore?

Reply:
Because you are supposed to call form helpers with symbols pointing to the classes attributes, they already translate it for you. Also this sets the `for` attribute of the label correctly so capybara can find the field if you give it the name of the field in the current locale. Some browser also provide convenience features based on the `for` attribute.